resources/lib/sosac.py

```diff
@@ -196,23 +206,22 @@ def list(self, url):
 
         return [self.dir_item(title="I failed", url="fail")]
 
+    def list_by_genres(self, url):
+        if "?" + GENRE_PARAM in url:
+            return self.list_xml_letter(url)
+        else:
+            result = []
+            page = util.request(url)
+            data = util.substr(page, '<select name=\"zanr\">', '</select')
```

Comment:
Mozno to neni tvoj kod, ale PR nesie tvoje meno. Problem je, ze ja osobne niesom ochotny toto mergnut ak to ostane v takomto stave. Takze niekto to musi prepisat a ja to zrejme nebudem.